index-handlers/src/main/java/no/sikt/nva/nvi/index/model/Approval.java

```diff
@@ -8,8 +8,8 @@
 @JacocoGenerated
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 @JsonSerialize
-public record Affiliation(String id,
-                          Map<String, String> labels,
-                          ApprovalStatus approvalStatus) {
+public record Approval(String id,
+                       Map<String, String> labels,
+                       no.sikt.nva.nvi.common.model.business.Status approvalStatus) {
```

Comment:
I would not use internal model for Status here. Create own index class for Status? Or convert it to a String in the index?